src/libs/ck-libs/ampi/CMakeLists.txt

```diff
@@ -167,22 +140,265 @@ configure_file(libmoduleampif.dep ${CMAKE_BINARY_DIR}/lib COPYONLY)
 
 
 # ROMIO
+set(romio-objects)
+set(romio-fortran-objects)
 if(${CMK_AMPI_WITH_ROMIO})
-    # Compile ROMIO with the same flags as AMPI
-    get_target_property(AMPI_CFLAGS moduleampi COMPILE_OPTIONS)
-    string(REPLACE ";" " " AMPI_CFLAGS "${AMPI_CFLAGS}")
-    set(ROMIO_FLAGS "${AMPI_CFLAGS} -ampi-romio-configure")
+    set(romio_dir ${CMAKE_BINARY_DIR}/src/libs/ck-libs/ampi/romio-prefix/src/romio)
+
+    if(CMK_WINDOWS)
+        set(obj_suf ".obj")
+    else()
+        set(obj_suf ".o")
+    endif()
+
+    set(romio-objects
+        ${romio_dir}/adio/ad_nfs/ad_nfs${obj_suf}
```

Comment:
Can you check - are the non-{nfs,ufs,testfs} filesystems not built ?

Reply:
Nevermind, seems like filesystems other than {nfs,ufs,testfs} need to be explicitly specified during ./configure.